### ISSUE TITLE
Is this dead?

### ISSUE THREAD

**Issue by Pemicope** [sc0ty/subsync#152]
Is this project dead? There hasn't been an update in a long time...

**Comment by rmatte** [comment]
I've seen recent pull requests merged, but it's not like this really needs updates all that frequently. Even with code from over a year ago, this is the only subtitle syncer I've found which works basically flawlessly. I've tried several others and none of them could sync subs as well as this does. I hope they do keep up development because this is hands down the best sub syncer out there.

**Comment by Pemicope** [comment]
> 

I agree with what you wrote. Personally I can't complain much, except sometimes it doesn't sync properly. Very rare but happens. The only real "complain" I have is that it doesn't work on Wayland in Linux. Other than that it works fine.
